pandas/io/formats/csvs.py

```diff
@@ -200,8 +247,16 @@ def _save_header(self):
 
         has_aliases = isinstance(header, (tuple, list, np.ndarray,
                                           ABCIndexClass))
-        if not (has_aliases or self.header):
-            return
+        if not (has_aliases or header):
+            # if index_label is False, nothing will display.
+            if index_label is False:
```

Comment:
Does it matter if this is None?

Reply:
it depends, if index label is not explicitly called which is `None`, index label is `True` if `header` or `index` is `not False`; otherwise, index label is set to `False`